examples/minimum_blending.py

```diff
@@ -10,6 +10,8 @@
 values, as opposed to the more conventional black [0, 0, 0]. For example, try the
 colormaps prefixed with *I*, such as *I Forest* or *I Bordeaux*, from 
 ChrisLUTs: https://github.com/cleterrier/ChrisLUTs .
+
+.. tags:: visualization-advanced
```

Comment:
```suggestion
.. tags:: visualization-basic
```

It's actually my example! 😬  
Forgot about tags, sorry!
Anyhow, it's rather basic—just setting a blending mode and colormap. Really only an example to show the visual effect.